mmocr/models/common/backbones/clip_resnet.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Tuple, Union
+
+import torch
+import torch.nn as nn
+from mmdet.models.backbones import ResNet
+from mmdet.models.backbones.resnet import Bottleneck
+
+from mmocr.registry import MODELS
+
+
+@MODELS.register_module()
+class AvgPool2d(nn.Module):
```

Comment:
Please add some docs, referring to https://github.com/open-mmlab/mmocr/blob/a09437adaa13a067c664928160222c46850534d3/mmocr/models/textrecog/plugins/common.py#L11

Reply:
Please add some UT, referring to https://github.com/open-mmlab/mmocr/blob/dev-1.x/tests/test_models/test_textrecog/test_plugins/test_maxpool.py

Reply:
done